iphone/TitaniumKit/TitaniumKit/Sources/API/TiUIView.m

```diff
@@ -363,7 +363,10 @@ - (void)setAccessibilityHint_:(id)accessibilityHint
 
 - (void)setAccessibilityHidden_:(id)accessibilityHidden
 {
-  self.accessibilityElementsHidden = [TiUtils boolValue:accessibilityHidden def:NO];
+  id accessibilityElement = self.accessibilityElement;
+  if (accessibilityElement != nil) {
```

Comment:
Is the nil check necessary here? Won't it be a noop anyway sending the below message to `accessibilityElement` if it's nil.

Also use the property style accessor `accessibilityElementsHidden `.

Reply:
If you see methods above this method, they use similar way. I think nil check was added to avoid to execute the other 3 statements (it is implemented previously). To make it more aligned with other methods (above), I have used method style accessor.

Reply:
Ah ok, that's alright then